provider-ui/package.json

```diff
@@ -27,9 +27,10 @@
     "styled-jsx": "latest"
```

Comment:
Can you edit the `npm test` script in `provider-ui/package.json` to run cypress with it?